app/src/main/java/com/codepath/apps/restclienttemplate/TweetsAdapter.java

```diff
@@ -0,0 +1,74 @@
+package com.codepath.apps.restclienttemplate;
+
+import android.content.Context;
+import android.text.Layout;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.codepath.apps.restclienttemplate.models.Tweet;
+
+import java.util.List;
+
+public class TweetsAdapter extends RecyclerView.Adapter<TweetsAdapter.ViewHolder> {
+
+    Context context;
```

Comment:
We want to avoid storing context in a field like this, as this can cause memory leaks. A better way is to get the local context from the system when needed.

https://medium.com/swlh/context-and-memory-leaks-in-android-82a39ed33002

Reply:
Thanks for the feedback! I will address this in a separate PR